packages/docs/docs/guides/3-API/3-api-apps.md

```diff
@@ -36,5 +39,6 @@ As an app developer, you will _not_ see your users' requests in your dashboard.
 :::
 
 
-# Get help
-TODO: Get in touch with us
\ No newline at end of file
+## Get help
+
+Do you have questions ? Or do you have troubles authenticating your app? Get in touch with us on the [Request Hub](https://requesthub.slack.com).
```

Comment:
I think you need to give an invite link, this is not enough

Reply:
```suggestion
Do you have questions ? Or do you have troubles authenticating your app? Get in touch with us on the [Request Hub](https://join.slack.com/t/requesthub/shared_invite/enQtMjkwNDQwMzUwMjI3LTc5NDRmN2YyMTVhZTBjNDE2MWU2YTBlYWIzYmJlYzNkMWQ5MzVmYzEzNGVmYjliNDQ4MjkyNTBiYjk4MDk3ZGE).
```